pkg/controller/smskeys/smskeys.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package smskeys contains web controllers for realm certificate key management.
+package smskeys
+
+import (
+	"context"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/cache"
+	"github.com/google/exposure-notifications-verification-server/pkg/config"
+	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/google/exposure-notifications-verification-server/pkg/keyutils"
+	"github.com/google/exposure-notifications-verification-server/pkg/render"
+)
+
+// Controller has handlers HTTP actions realted to SMS signing key management.
+type Controller struct {
+	config         *config.ServerConfig
+	db             *database.Database
+	h              render.Renderer
+	publicKeyCache *keyutils.PublicKeyCache
+}
+
+// New creates a new Controller
+func New(ctx context.Context, config *config.ServerConfig, db *database.Database, cacher cache.Cacher, h render.Renderer) (*Controller, error) {
```

Comment:
Consider passing in the public key cache as a parameter instead of building it in the controller. That removes the need to pass in a context to New and removes the need to return an error.

Reply:
did same for realmkeys